dart/lib/src/hub.dart

```diff
@@ -470,6 +466,7 @@ class Hub {
       }
 
       transactionContext.origin ??= SentryTraceOrigins.manual;
+      transactionContext.traceId = scope.propagationContext.traceId;
```

Comment:
navigator observer will take care of updating the propagation context so new transactions just need to use it